components/d2l-work-to-do/d2l-work-to-do.js

```
@@ -493,6 +493,18 @@ class WorkToDoWidget extends EntityMixinLit(LocalizeWorkToDoMixin(LitElement)) {
 		}
 	}
 
+	// !TODO Further investigate defect DE42208 to remove need for retry
+	async _performSirenActionWithRetry(token, action, fields, immediate, maxRetries = 0, error = null)  {
+		if (maxRetries > -1) {
+			return performSirenAction(token, action, fields, immediate)
```

I just want to make sure, but does this actually throw an error? I thought these swallowed the error and spit out undefined?

---

Verified that in this case an error is actually thrown and it shows in console in the unhandled case.

---

Thanks Surag. That must've been a long one to verify.